load_testing/lib/flow_sign_up.py

```diff
@@ -69,8 +70,23 @@ def do_sign_up(context):
         },
     )
 
+    auth_token = authenticity_token(resp)
+
+    resp = do_request(
+        context,
+        "post",
+        "/authentication_methods_setup",
+        "/phone_setup",
+        "",
+        {
+            "_method": "patch",
+            "two_factor_options_form[selection][]": "",
+            "two_factor_options_form[selection][]": "phone",
```

Comment:
```suggestion
            "two_factor_options_form[selection][]": "phone",
```
duplicate key?

Reply:
I don't think it's strictly needed since we're specifying one here, but the inclusion is intentional to ensure the form includes at least an empty value when none are selected ([StackOverflow](https://stackoverflow.com/questions/12084909/rails-forms-when-no-checkboxes-are-checked-param-is-not-included-in-post))

https://github.com/18F/identity-idp/pull/6471/files#diff-82fe2b4bb7ac5a020ce914055a9f7920e33ba54da091cc0e8130d1abb5f77982R14

Reply:
yeah, but in python, with two of the same key in the same hash, only one gets gets actuall sent over the wire. We'd need like an array syntax to capture that blank value in python `[['key', ''], ['key', 'phone']]`

```python
# python3
>>> x = {
... 'a': '',
... 'a': 'second_value',
... }
>>> x
{'a': 'second_value'}
```

Reply:
oh, of course, thank you, will fix 🤦🏼‍♂️